scalable-integrator/volumes/persistent-volumes.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 kind: PersistentVolume
 metadata:
-  name: shared-persistent-volume
+  name: integrator-server-share-persistent-volume
```

Comment:
Shall we rename this to "scalable-integrator-shared-persistent-volume" ?

Reply:
+1